test cases/common/282 override version/meson.build

```diff
@@ -0,0 +1,15 @@
+project('override version', 'c',
+  version : '3.14')
+
+app = executable('app', 'app.c')
+
+meson.override_find_program('mesontest-app', app)
+
+found_app = find_program('mesontest-app')
+
+if found_app.version() != meson.project_version()
+  error('Overriden Executable had incorrect version: got @0@, expected @1@'.format(found_app.version(), meson.project_version()))
+endif
```

Comment:
In line with treating this as a bugfix, I'd just add the test to the existing `test cases/native/9 override with exe`.